src/css/components/_control.scss

```diff
@@ -1,4 +1,5 @@
-.video-js .vjs-control {
+.video-js .vjs-control,
+.video-js .vjs-button {
```

Comment:
Actually, we can remove all of these vjs-button additions in the CSS. All of the elements already have the `vjs-control` class name, so all of those styles will already apply.